help/destinations/catalog/advertising/tradedesk-emails-update

```diff
@@ -0,0 +1,138 @@
+
+You can hash email addresses before ingesting them into Adobe Experience Platform or use raw email addresses. 
+
+To learn about ingesting email addresses in Experience Platform, see the [batch ingestion overview](https://experienceleague.adobe.com/docs/experience-platform/ingestion/batch/overview.html?lang=en). 
+
+If you select to hash the email addresses yourself, make sure to comply with the following requirements: 
+
+*  Remove leading and trailing spaces. 
+*  Convert all ASCII characters to lowercase. 
+*  In `gmail.com` email addresses, remove the following characters from the username part of the email address: 
+      *  The period (. (ASCII code 46)). For example, normalize `jane.doe@gmail.com` to `janedoe@gmail.com`. 
+      *  The plus sign (+ (ASCII code 43)) and all subsequent characters. For example, normalize `janedoe+home@gmail.com` to `janedoe@gmail.com`. 
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (email or hashed email) used in the Trade Desk destination. |
+| Export frequency | **[!UICONTROL Daily Batch]** | As a profile is updated in Experience Platform based on segment evaluation, the profile (identities) are updated once a day downstream to the destination platform. Read more about [batch uploads](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-types.html?lang=en#file-based).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+### Authenticate to Destination {#authenticate}
+
+[!DNL The Trade Desk] CRM Destination is a daily batch file upload and does not require authentication by the user.
+
+### Fill in Destination Details {#fill-in-details}
+
+Before you can send, or activate, audience data to a destination, you must set up a connection to your own destination platform. While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+
+*  **[!UICONTROL Account Type]**: Please choose the **[!UICONTROL Existing Account]** option. 
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future. 
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Advertiser ID]**: your [!DNL Trade Desk Advertiser ID], which can either be shared by your [!DNL Trade Desk] Account Manager or be found under [!DNL Advertiser Preferences] in the [!DNL Trade Desk] UI. 
+
+![Platform UI screenshot showing how to fill in destination details] (help/destinations/assets/catalog/advertising/tradedesk/configuredestination2.png)
```

Comment:
Please update the link to start with an `/`, otherwise it won't render. So, the correct link would be `/help/destinations/assets/catalog/advertising/tradedesk/configuredestination2.png`

Also, please make sure that there is no space between the closing `]` and opening `(`. Should be like: `details](/help`

The comment above applies to all linked images.

Reply:
Updated.